src/components/base/outline-tooltip/outline-tooltip.css

```diff
@@ -0,0 +1,41 @@
+:host {
+  @apply inline-flex flex-col;
+}
+
+.tooltip {
+  @apply flex relative justify-center;
+}
+
+.tooltip-tip {
+  @apply absolute inline-flex p-2 opacity-0 text-center;
+  visibility: hidden;
```

Comment:
we can add `hidden` to the `@apply` directive to accomplish the `visibility: hidden` property here

Reply:
No. They work differently. Hidden = display: 'none'; Which is great but it means you can't use transitions. Also I'm using the visibility css attribute to control the cursor events, so that you won't know the tip is there and can't trigger it or anything in it until the visibility is set to visible.

Reply:
Gotcha! Thanks for clearing that up for me